CHANGELOG.md

```diff
@@ -1,5 +1,111 @@
 # Changelog
 
+## v6.0.0-beta.8 - 2017-10-05
+
+### Engine updates and fixes
+
+* Make calls to void CodeMethod work (#4850) (Thanks @powercode)
+* Get 'PSVersion' and 'GitCommitId' from the 'ProductVersion' attribute of assembly (#4863) (Thanks @iSazonov)
+* Write an error and return an exit code for powershell -version 2 (#4931) (Thanks @iSazonov)
```

Comment:
This code was essentially replaced by #4958, perhaps just combine them and attribute to Ilya

Reply:
resolved